streamz/core.py

```diff
@@ -447,6 +447,49 @@ def update(self, x, who=None):
             return []
 
 
+@Stream.register_api()
+class starsink(Stream):
+    """ Apply a function on every element
+
+    Examples
+    --------
+    >>> source = Stream()
+    >>> L = list()
+    >>> source.sink(L.append)
```

Comment:
The example here doesn't seem to demonstrate starsink at all.

Reply:
Sorry, too much copy paste :sheep: